charts/ndm/templates/02-daemonset.yaml

```diff
@@ -0,0 +1,164 @@
+{{- if .Values.ndm.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "ndm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "ndm.name" . }}
+    chart: {{ template "ndm.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
What is the heritage label used for?

Reply:
I am not sure. I saw it on the previous charts and hence added it.

Reply:
The heritage label value is `helm` in this case. It signifies the package manager used to deploy the artifacts.